boards/BMS/Inc/App/App_Imd.h

```diff
@@ -17,7 +17,7 @@ enum Imd_ConditionName
     IMD_UNDERVOLTAGE_DETECTED,
     IMD_SST,
     IMD_DEVICE_ERROR,
-    IMD_EARTH_FAULT,
+    IMD_GROUND_FAULT,
```

Comment:
I mostly changed this out of petty-ness, if "Earth" is actually the standard for this kind of thing I'll change it back